src/scportrait/pipeline/_utils/sdata_io.py

```diff
@@ -125,17 +169,25 @@ def _write_segmentation_object_sdata(
         _sdata.write_element(segmentation_label, overwrite=True)
 
         self.log(f"Segmentation {segmentation_label} written to sdata object.")
-
         self._check_sdata_status()
 
     def _write_segmentation_sdata(
         self,
-        segmentation,
+        segmentation: xarray.DataArray,
```

Comment:
this can also be a numpy array would we then specify this via xarray.DataArray | np.ndarray?

Reply:
Yes. Done.